lib/membrane_audio_interleaver.ex

```diff
@@ -69,13 +69,7 @@ defmodule Membrane.AudioInterleaver do
   end
 
   @impl true
-  def handle_pad_added(_pad, %{playback_state: playback_state} = _context, _state)
-      when playback_state != :stopped do
-    raise("All pads should be connected before starting the element!")
-  end
```

Comment:
I obviously I've missed the `!` when making a suggestion and you handled that well. However, this explicit error seems like a good addition. You can add it after the existing clause